offer-manager-domain/src/main/groovy/life/qbic/business/products/Converter.groovy

```diff
@@ -108,4 +109,15 @@ class Converter {
     static Product convertProductToDTO(life.qbic.business.products.Product product){
         return createProductWithVersion(product.category,product.name, product.description, product.unitPrice, product.unit, product.id.uniqueId)
     }
+
+    /**
+     * Creates a duplicate product based on the provided product and the given ID
+     * @param product Product to be duplicated
+     * @param newId The new product id of the duplicated product
+     * @return a product with the information form the provided product and the given product id
+     */
+    static Product duplicateProduct(Product product, ProductId newId){
+        ProductCategory category = getCategory(product)
+        return createProductWithVersion(category,product.productName,product.description,product.unitPrice,product.unit,newId.uniqueId)
+    }
```

Comment:
This public method looks to me like boiler plate code. Why is it needed here in the domain package?

Reply:
This was a suggestion from @sven1103 for another PR where I initially needed to duplicate a product (this functionality was then moved to the DB and thus, this PR)

Reply:
I will rethink the design 👍🏼

Reply:
I removed the method